common/opinion_modifiers/hfe_opinion_features.txt

```diff
@@ -1,22 +1,22 @@
 #
 
 # same class
-hfe_opinion_features_same_species_class = {
-	trigger = {
-		hfe_trigger_features_opinion_enabled = yes
-		is_same_species_class = from
-		is_homicidal = no
-		from = {
-			is_homicidal = no
-		}
-		not = {
-			is_same_species = from
-		}
-	}
-	opinion = {
-		base = 5
-	}
-}
+#hfe_opinion_features_same_species_class = {
```

Comment:
I do not see why it should be removed

Reply:
It produced a butt ton of errors in the CW Tools Validator and the Paradox Error Logs.

Reply:
What errors?